### DISCUSSION TITLE
Bundle size

### DISCUSSION THREAD

**Discussion by renet** [floating-ui/floating-ui#1340] [General]
According to bundlephobia the minified and gzipped bundle size is no longer ~3 kB, as the home page prominently states, but grew to [6.9 kB](https://bundlephobia.com/package/@popperjs/core) lately. 🤭

Am I missing something? If not, maybe you should update your docs. ;)

**Comment by FezVrasta** [comment]
Bundlephobia reports a wrong size, their computation includes all the webpack bundle utilities.

The size advertised on the home page refers to the minimal version of the library.

You can find real-time updated badges on the bottom of this page:
https://popper.js.org/docs/v2/#distribution-targets
